python/bridge/default_debug_bridge.py

```diff
@@ -287,6 +288,50 @@ def gdb(self, port):
         self.gdb_handle = self.module.gdb_server_open(self.get_cable().get_instance(), port)
         return 0
 
```

Comment:
Could you move this command to the gap class as it is only relevant for gap. You can keep an empty command in this default class reporting an error